package.json

```diff
@@ -35,15 +36,18 @@
     "eslint-plugin-jsx-a11y": "^6.0.2",
     "eslint-plugin-react": "^7.4.0",
     "eslint-watch": "^3.1.3",
+    "extract-text-webpack-plugin": "^4.0.0-beta.0",
+    "fsevents": "^2.1.3",
```

Comment:
These don't seem to be used. If they are not, please remove them as well.

Reply:
ok